### DISCUSSION TITLE
浩辰cad重新激活  原链接失效求博主再发一份

### DISCUSSION THREAD

**Discussion by kegongzhudi** [AlistGo/alist#657] [General]
![Screenshot_20220303_004444_com huawei browser](https://user-images.githubusercontent.com/100811886/156476302-75c3e448-ede1-4578-be34-b8c9b80de2ac.png)
原链接：https://zhixiangban.net/89.html
万分感谢博主

**Comment by xhofe** [comment]
...我不知道你怎么找过来的，但这不是我的网站。
